tests/compose/fixtures/v3-full/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: "3.2"
```

Comment:
according to here https://github.com/vmware/vic-product/blob/master/docs/user_doc/vic_app_dev/docker_compose_file_options.md#supported-docker-compose-file-options, currently we only support compose file version 2 and 2.1. I don't know when exactly we will claim supporting for 3.2